### PR TITLE
[MRG] raw_to_bids: allow adding non-maxfiltered data

### PR DESCRIPTION
**Pull request by dnacombo**
Merge checklist
---------------

Maintainer, please confirm the following before merging:

- [x] All comments resolved
- [x] This is not your own PR
- [x] All CIs are happy
- [x] PR title starts with [MRG]
- [x] [whats_new.rst](https://github.com/mne-tools/mne-bids/blob/master/doc/whats_new.rst) is updated
- <s>[ ] PR description includes phrase "closes <#issue-number>"</s>
- [x] Commit history does not contain any merge commits

### CODE REVIEW

**Comment by jasmainak** [comment]
Thanks for the PR. Could you put back the checklist you were presented with when you made the PR?

Also, two things. Your commit email address is not linked to your github id. It would be nice if you used the same. You need to update whats_new. Otherwise, PR looks good.

File: mne_bids/io.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def _parse_ext(raw_fname, verbose=False):
 
 
 def _read_raw(raw_fname, electrode=None, hsp=None, hpi=None, config=None,
-              montage=None, verbose=None):
+              montage=None, verbose=None, allow_maxshield=None):
```
Comment (jasmainak) [inline mne_bids/io.py]: why None? isn't this supposed to be boolean? Let's use False by default as in MNE?

**Comment by dnacombo** [comment]
> Thanks for the PR. Could you put back the checklist you were presented with when you made the PR?
I don't know what you're talking about. I was not using the github interface to send the PR. I'm using a gui (smartgit) that allows doing that.

> Also, two things. Your commit email address is not linked to your github id. It would be nice if you used the same. 
Ok I added the missing email to github.
> You need to update whats_new. Otherwise, PR looks good.
Ok. First PR: check! :)

**Comment by jasmainak** [comment]
> I don't know what you're talking about. I was not using the github interface to send the PR. I'm using a gui (smartgit) that allows doing that.

okay, seems like it's too smart. I [edited the comment](https://github.com/mne-tools/mne-bids/pull/171#issue-255989712) for you.

one more thing. `allow_maxshield` should be False by default, not `None`

File: mne_bids/commands/mne_bids_raw_to_bids.py [hunk 2]
```diff
@@ -54,14 +54,18 @@ def run():
     parser.add_option('--overwrite', dest='overwrite',
                       help=("Boolean. Whether to overwrite existing data"),
                       metavar='ow')
+    parser.add_option('--allow_maxshield', dest='allow_maxshield',
+                      help=("Boolean. Whether to allow non Maxfiltered data."),
+                      metavar='mxf')
```
Comment (jasmainak) [inline mne_bids/commands/mne_bids_raw_to_bids.py]: does this actually work? What command do you use? I think you may have to do `action="store_true"` or something to that effect

Comment (dnacombo) [reply mne_bids/commands/mne_bids_raw_to_bids.py]: Works like a charm.
`mne_bids raw_to_bids --subject_id sub01 --task rest --raw emo01.fif --output_path ./study --allow_maxshield True`
image

Comment (jasmainak) [reply mne_bids/commands/mne_bids_raw_to_bids.py]: The problem with this approach is that user needs to specify `True`. What if they provided `true` instead? The command line approach is to do:

mne_bids raw_to_bids --subject_id sub01 --task rest --raw emo01.fif --output_path ./study --allow_maxshield

It's similar to what you do with a parameter like verbose

File: doc/whats_new.rst [hunk 3]
```diff
@@ -29,6 +29,7 @@ Bug
 API
 ~~~
 
+- Add support for non maxfiltered .fif files, by `Maximilien Chaumon`_ (`#171 <https://github.com/mne-tools/mne-bids/pull/171>`_)
```
Comment (jasmainak) [inline doc/whats_new.rst]: Your name will not render with a link unless you also provide a website down below like the others.

Comment (dnacombo) [reply doc/whats_new.rst]: fine with me. Thanks

Comment (jasmainak) [reply doc/whats_new.rst]: I meant that you need to add the weblink at the bottom. You can give your github weblink. Or alternatively, remove the markup for links.

**Comment by jasmainak** [comment]
@dnacombo after fixing the whats_new page, can you add [MRG] to PR title when you are ready for it to be merged? Thanks

**Comment by agramfort** [comment]
@dnacombo you should not open PRs from your master branch

**Comment by dnacombo** [comment]
> @dnacombo you should not open PRs from your master branch

Now I know why! didn't know updates to the branch were automatically added to the PR.

**Comment by jasmainak** [comment]
Merged. Thanks for your first PR to mne-bids @dnacombo !

File: mne_bids/io.py [hunk 4]
```diff
@@ -63,7 +63,7 @@ def _read_raw(raw_fname, electrode=None, hsp=None, hpi=None, config=None,
                               preload=False, verbose=verbose)
 
     elif ext in ['.fif', '.ds', '.vhdr', '.set']:
-        raw = reader[ext](raw_fname)
+        raw = reader[ext](raw_fname, allow_maxshield=allow_maxshield)
```
Comment (jasmainak) [inline mne_bids/io.py]: ufff looked like I merged this PR hastily. The option `allow_maxshield` does not apply to all readers, only for fif readers. @dnacombo can you send a PR to fix this? Thanks

**Comment by dnacombo** [comment]
Out of town for a week. Will look into it after that.

**Comment by jasmainak** [comment]
okay never mind then. I'll fix it in my PR #135

**Comment by dnacombo** [comment]
Ok. Thanks!

**Pull request merged**